src/main/java/lotto/domain/LottoTicket.java

```diff
@@ -0,0 +1,56 @@
+package lotto.domain;
+
+import java.util.ArrayList;
+import java.util.Collections;
+import java.util.List;
+import java.util.stream.Collectors;
+import java.util.stream.IntStream;
+
+public class LottoTicket {
+    public static final int PRICE = 1000;
+    public static final int MIN_NUMBER = 1;
+    public static final int MAX_NUMBER = 45;
+    public static final int LOTTO_SIZE = 6;
+    private final List<Integer> lottoNumbers;
```

Comment:
```suggestion
    public static final int LOTTO_SIZE = 6;
    
    private final List<Integer> lottoNumbers;
```

상수와 멤버변수를 구분 짓는 것이 저는 보기 좋더라구요 😄

Reply:
감사합니다. 반영했습니다.